charts/xui-webapp/values.yaml

```diff
@@ -55,7 +55,8 @@ nodejs:
     SERVICES_CCD_CASE_ASSIGNMENT_API: http://aac-manage-case-assignment-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     SERVICES_LOCATION_API: http://rd-location-ref-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     # Services Hearings endpoints
-    SERVICES_HEARINGS_COMPONENT_API: http://jurisdiction-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    SERVICES_SSCS_HEARINGS_COMPONENT_API: http://sscs-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
This is the trickiest bit to make generic. I'd like to get rid of these effectively duplicate values, but it's not easy
 
 it's a YAML file, so we can do this
SERVICE_HEARINGS_COMPONENT_API: 
  sscs: http://sscs-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
  privatelaw: http://wibble.wobble-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal

Then reference it in variables.json as SERVICE_HEARINGS_COMPONENT_API.sscs 

Which is slighly better